src/controller/python/chip/native/api_define.py

```diff
@@ -0,0 +1,11 @@
+from ctypes import *
+from chip.exceptions import ChipStackError
+from dataclasses import dataclass
+from typing import List, Callable
+
+
+@dataclass
+class ApiDefine:
```

Comment:
Don't really understand what this class does.

Reply:
This function no longer exists